container_registry/container_analysis/README.md

```diff
@@ -0,0 +1,59 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
+Cloud Platform logo" title="Google Cloud Platform" align="right" height="96"
+width="96"/>
+
+# Google Cloud Container Analysis Samples
+
+
+Container Analysis scans container images stored in Container Registry for vulnerabilities.
+Continuous automated analysis of containers keep you informed about known vulnerabilities so 
+that you can review and address issues before deployment.
+
+Additionally, third-party metadata providers can use Container Analysis to store and 
+retrieve additional metadata for their customers' images, such as packages installed in an image.
+
+
+## Description
+
+These samples show how to use the [Google Cloud Container Analysis Client Library](https://cloud.google.com/container-registry/docs/reference/libraries).
+
+## Build and Run
+1.  **Enable APIs** - [Enable the Container Analysis API](https://console.cloud.google.com/flows/enableapi?apiid=containeranalysis.googleapis.com)
+    and create a new project or select an existing project.
+1.  **Install and Initialize Cloud SDK**
+    Follow instructions from the available [quickstarts](https://cloud.google.com/sdk/docs/quickstarts)
+1.  **Clone the repo** and cd into this directory
+
+```
+    $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples
+    $ cd python-docs-samples/container-registry/container-analysis
```

Comment:
Should be `cd python-docs-samples/container_registry/container_analysis/`

(underscores, not hyphens)

Reply:
Wow good catch! I copy/pasted from the ruby README, must have missed that